client/src/components/atoms/NotificationListItem.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <q-item clickable :class="{ unread: !note.viewed }" class="q-pl-none">
+    <q-badge v-if="!note.viewed" />
+    <q-item-section side class="q-px-md">
+      <q-icon :size="iconSize" :name="note.icon" />
```

Comment:
I've noticed that the color of these icons does not have a sufficient color contrast when they're against the blue background highlight signifying the notification is unread. Consider making these darker when they're for unread notifications?

Reply:
This is a size issue and not a color contrast issue.  Fixed.